docs/user/discover.asciidoc

```diff
@@ -243,7 +243,7 @@ the table columns that display by default, and more.
 
 --
 
-include::{kib-repo-dir}/discover/context.asciidoc[]
+include::{kib-repo-dir}/discover/view-document.asciidoc[]
```

Comment:
Since the file is being renamed, you'll need to put a redirect in redirects.asciidoc.

Reply:
Added it, thx, how can I test this?